GDJS/Runtime/runtimeobject.js

```diff
@@ -318,6 +318,7 @@ gdjs.RuntimeObject.prototype.deleteFromScene = function(runtimeScene) {
  */
 gdjs.RuntimeObject.prototype.onDestroyFromScene = function(runtimeScene) {
     var theLayer = runtimeScene.getLayer(this.layer);
+
```

Comment:
You'll need to adapt every place where `getRendererObject`/`getRenderer` is used to optionally do the same with the normal map object. This is: `onDestroyFromScene`, `setZOrder`, `setLayer`.